### PR TITLE
[ADD] document_page_tags

### PR DESCRIPTION
**Pull request by hbrunn**
# 
# Tags/Keywords for document page

This module allows you to fill in keywords on your pages to simplify finding them afterwards.
# Usage

Fill in your keywords, and search for them by typing (part of) the keyword and choose searching for keywords.

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:

File: document_page_tags/models/document_page_tag.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+##############################################################################
+from openerp import models, fields
+
+
+class DocumentPageTag(models.Model):
+    _name = 'document.page.tag'
+    _description = 'A keyword for document pages'
+
+    name = fields.Char()
```
Comment (StefanRijnhart) [inline document_page_tags/models/document_page_tag.py]: Maybe make this field translatable, as are other tag objects in the core (res.partner.category and crm.case.categ for instance)?

**Comment by hbrunn** [comment]
@StefanRijnhart thanks, done

**Comment by andhit-r** [comment]
@hbrunn no menu to manage tags?

**Comment by hbrunn** [comment]
@andhit-r would be nice I guess. we use this in production for months by now and don't muss it, so I won't put extra effort in it, but if you'd like to PR it into this branch, that would be most welcome

**Comment by StefanRijnhart** [comment]
Thanks for the update! :+1:

**Comment by lfreeke** [comment]
:+1: (test)

**Pull request merged**